draft-song-dns-wireformat-http-01.xml

```diff
@@ -371,6 +398,12 @@
     	<t>Registration for a new HTTP header field: Proxy-DNS-Transport</t>
     
```

Comment:
I do not think there is any requirement that a IETF standards-track document is necessary for a HTTP header field. (Certainly in DNS this is not required.)